packages/intl-locale/index.ts

```diff
@@ -640,44 +640,90 @@ export class Locale {
   }
 
   /**
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getCalendars
+   * https://tc39.es/proposal-intl-locale-info/#sec-Intl.Locale.prototype.getCalendars
+   */
+  public getCalendars() {
+    return calendarsOfLocale(this)
+  }
+
+  /**
+   * @deprecated – use local.getCalendars() instead
```

Comment:
this is not deprecated. Same as the ones below

Reply:
According the MDN:

> Note: In some versions of some browsers, this method was implemented as an accessor property called `calendars`. However, because it returns a new array on each access, it is now implemented as a method to prevent the situation of locale.calendars === locale.calendars returning false. Check the browser compatibility table for details.

This seems to imply that using the getter `.calendar` is deprecated

Reply:
The spec doesn't say it's deprecated

Reply:
They fully removed every mention of the old getters (on https://tc39.es/proposal-intl-locale-info, the getter `.calendars` is not referenced anymore, just `.getCalendars` is)

I only kept the getters to avoid introducing a major change and to be able to tell users of the lib that they should switch to the new attributes

Reply:
My previous point still stands. Spec doesn't say deprecated anywhere.

Reply:
What are you recommending me to do on this PR?
1. Keep both `.getCalandars` and `.calendars` as it is at the moment
2. Only keep the new attributes and remove the getter ones (to follow the new spec)
3. Remove the deprecation notice but keep the rest unchanged (2 attributes with the getters calling the other ones)
4. Keep both without calling `this.getCalendars()` in `get calendars` (and duplicating the logic)
5. Only keep the getters and removing the new methods (aka closing this PR)

Reply:
4 sounds like the most spec compliant option so I'd recommend that

Reply:
None of the other options would pass test262

Reply:
Updated